Katello/katello
# Fixes #37678 - Add postgresql-evr as a migration

#### What are the changes introduced in this pull request?
Early draft that works for new setups.
Need to work out existing installations.
#### Considerations taken when implementing this change?
We want to drop the postgres_evr extension and move all that to live inside katello.

#### What are the testing steps for this pull request?
1. Check `sudo rpm -qa *evr*`
2. Remove the rpm: `sudo rpm -e evr_package_from_above`
3. Run `bundle exec rails katello:reset` to mimic fresh installs
4. Test the usual flows with applicability

To test existing systems just delete the evr package and make sure everything continues to work as normal.
